MozillaFoundation/foundation.mozilla.org
# Allow homepage sections to be editable.

# Description

This PR enables customization for the homepage section titles `News you can use` being renamed as `The Highlights`, `Ideas`, and `Take action`.

<img width="942" alt="image" src="https://github.com/user-attachments/assets/bc4bc987-6333-43a9-a8aa-9007d947086f">

<img width="956" alt="image" src="https://github.com/user-attachments/assets/09315c27-ef34-45b4-bde8-f56c660cc21e">

<img width="975" alt="image" src="https://github.com/user-attachments/assets/bd78960b-2c1e-4e28-9430-08d84afa8002">

# To test

1. Log in to the [review app CMS](https://foundation-s-tp1-973-al-9h9u2r.herokuapp.com/cms/) with credentials `admin2:admin2`
2. [Edit the homepage](https://foundation-s-tp1-973-al-9h9u2r.herokuapp.com/cms/pages/46/edit/). Update the `Highlights`, `Ideas` and `Take action` title fields. Publish.
3. Open the [review app homepage](https://foundation-s-tp1-973-al-9h9u2r.herokuapp.com/en/) and confirm changes were applied.

Link to sample test page: https://foundation-s-tp1-973-al-9h9u2r.herokuapp.com/en/
Related PRs/issues: #12601

┆Issue is synchronized with this [Jira Story](https://mozilla-hub.atlassian.net/browse/TP1-1219)
